lib/util.py

```diff
@@ -632,10 +632,19 @@ def check_www_dir(rdir):
         src = os.path.join(os.path.dirname(__file__), 'www', 'index.html')
         shutil.copy(src, index)
     files = [
-        "https://code.jquery.com/jquery-1.9.1.min.js",
-        "https://raw.githubusercontent.com/davidshimjs/qrcodejs/master/qrcode.js",
-        "https://code.jquery.com/ui/1.10.3/jquery-ui.js",
-        "https://code.jquery.com/ui/1.10.3/themes/smoothness/jquery-ui.css"
+        "https://code.jquery.com/jquery-3.2.1.min.js"
+        "https://raw.githubusercontent.com/neocotic/qrious/master/dist/umd/qrious.min.js",
```

Comment:
This returns a 404 error. Also, AFAIK, using `raw.githubusercontent.com` in production is discouraged, because it's not very efficient. Also, they send the files with `text/plain` mime type, which will cause some browsers to not execute them as JS

Reply:
It's not needed to execute these files, obviously, they are downloaded just once. 

IMHO it is needed to include copies of these files into Electrum repository to be sure they are correctly distributed. 

What do you think about it, guys?

Reply:
I would prefer this to be hosted in a separate project. will look into how to do this.

Reply:
Good idea, I definitely can co-work on this project, as I use it every day. Let me know what have you decided.